src/Mvc/Mvc.Core/src/ConsumesAttribute.cs

```diff
@@ -96,6 +97,11 @@ public ConsumesAttribute(Type requestType, string contentType, params string[] o
         /// </summary>
         public MediaTypeCollection ContentTypes { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value that determines if the request body is required or not.
```

Comment:
```suggestion
        /// Gets or sets a value that determines if the request body is optional. This value is only used to specify if the request body is required in API explorer.
```

I'd like to make sure we clarify this is informational and does not affect MVC's binding behavior since there's 2 separate ways to configure required body binding.